bigbluebutton-html5/imports/ui/components/actions-bar/create-breakout-room/component.jsx

```diff
@@ -160,15 +175,42 @@ class BreakoutRoom extends Component {
     Session.set('isUserListOpen', true);
   }
 
+  onInviteBreakout() {
+    const { getBreakouts, sendInvitation } = this.props;
+    const { users } = this.state;
+    const breakouts = getBreakouts();
+    if (users.length === this.getUserByRoom(0).length) {
+      this.setState({ valid: false });
+      return;
+    }
+
+    users
+      .filter(u => u.room > 0)
+      .forEach((user) => {
+        const breakout = breakouts[user.room - 1];
```

Comment:
`breakouts[user.room - 1]` -- is this 100% reliable - could we not filter by an identifier?